analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -209,7 +209,9 @@ def uglify(text):
             rows = context.available_profiles.items()
         else:
             header = ['Profile name']
-            rows = ([key] for key in context.available_profiles.keys())
+            rows=[]
```

Comment:
You can insert a tuple in the short array creation syntax the following way:
```suggestion
            rows = ([(key, "")] for key in context.available_profiles.keys())
```

Reply:
I fixed as you suggested + added a test case